documentation/docs/scenarios/xsa-deploy/Readme.md

```diff
@@ -0,0 +1,96 @@
+# Build and Deploy SAPUI5 or SAP Fiori Applications on SAP Cloud Platform with Jenkins
+
+Build an application based on SAPUI5 or SAP Fiori with Jenkins and deploy the build result into an SAP Cloud Platform account in the Neo environment.
+
+## Prerequisites
+
+* You have installed the Java Runtime Environment 8.
+* You have installed Jenkins 2.60.3 or higher.
+* You have set up Project “Piper”. See [README](https://github.com/SAP/jenkins-library/blob/master/README.md).
+* You have installed the Multi-Target Application (MTA) Archive Builder 1.0.6 or newer. See [SAP Development Tools](https://tools.hana.ondemand.com/#cloud). TODO: maybe we should refer to the docker image
+* You have installed Node.js including node and npm. See [Node.js](https://nodejs.org/en/download/). TODO: docker ?
+
+### Project Prerequisites
+
+This scenario requires additional files in your project and in the execution environment on your Jenkins instance.
+
+On the project level, provide and adjust the following template:
+
+| File Name | Description | Position |
+|-----|-----|-----|
+| [`.npmrc`](https://github.com/SAP/jenkins-library/blob/master/documentation/docs/scenarios/ui5-sap-cp/files/.npmrc) | This file contains a reference to the SAP NPM registry (`@sap:registry https://npm.sap.com`), which is required to fetch the dependencies required to build the application. | Place the `.npmrc` file in the root directory of your project. |
```

Comment:
```suggestion
| [`.npmrc`](https://github.com/SAP/jenkins-library/blob/master/documentation/docs/scenarios/ui5-sap-cp/files/.npmrc) | This file contains a reference to the SAP NPM registry (`@sap:registry https://npm.sap.com`), which you need to fetch the dependencies required to build the application. | Place the `.npmrc` file in the root directory of your project. |
```

Reply:
@SarahNoack your comment is outdated. We do not describe technology specific files here again, we rely solely on `mtaBuild` and `xsDeploy`. What we build through `mtaBuild` is in the meantime considered to be out of scope.